app/Controllers/LibraryThingImportController.php

```
@@ -77,15 +77,15 @@ private function hasDescrizionePlainColumn(\mysqli $db): bool
+     * Route LibraryThing import diagnostics through SecureLogger so PII
+     * redaction + retention apply (CR R7). The previous flat-file sink
+     * logged raw ISBNs, titles, and stack traces with no rotation.
      */
     private function log(string $message): void
     {
-        $logFile = __DIR__ . '/../../writable/logs/import.log';
-        $timestamp = date('Y-m-d H:i:s');
         // Sanitize message to prevent log injection (strip newlines/control chars)
         $message = str_replace(["\r", "\n", "\t"], ' ', $message);
-        file_put_contents($logFile, "[$timestamp] [LT] $message\n", FILE_APPEND);
+        \App\Support\SecureLogger::info('[LT] ' . $message);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SecureLogger' in method 'log'. (undefined)

(StaticAccess)